content/en/docs/concepts/security/index.md

```diff
@@ -630,333 +631,277 @@ runtime. When a request comes to the proxy, the authorization engine evaluates
 the request context against the current authorization policies, and returns the
 authorization result, `ALLOW` or `DENY`.
 
-### Enabling authorization
+### Implicit Enablement
 
-You enable Istio Authorization using a `ClusterRbacConfig` object. The `ClusterRbacConfig`
-object is a cluster-scoped singleton with a fixed name value of `default`. You can
-only use one `ClusterRbacConfig` instance in the mesh. Like other Istio configuration
-objects, `ClusterRbacConfig` is defined as a
-Kubernetes `CustomResourceDefinition`
-[(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) object.
+There is no need to explicitly enable Istio authorization feature, you just apply
```

Comment:
There is no need to explicitly enable Istio's authorization feature, you just apply